qiskit_nature/circuit/library/ansaetze/evolved_operator_ansatz.py

```diff
@@ -0,0 +1,135 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+"""The evolved operator ansatz."""
+
+from typing import List, Union, Optional
+
+from qiskit.circuit import Parameter, ParameterVector, QuantumRegister
+from qiskit.circuit.library import BlueprintCircuit
+from qiskit.opflow import OperatorBase, EvolutionBase, PauliTrotterEvolution
+
+
+class EvolvedOperatorAnsatz(BlueprintCircuit):
+    """The evolved operator ansatz."""
+
+    def __init__(self,
+                 operators: Union[OperatorBase, List[OperatorBase]],
```

Comment:
Should this be optional to allow them to be passed later and maybe default to None. Currently the code checks if they are still None but neither here nor the setter state that it is Optional